content/blog/adoptopenjdk-16-available/index.md

```diff
@@ -0,0 +1,52 @@
+
+For a complete list of the enhancements (including ones that only impact developers of OpenJDK), [see the JDK 16 overview over at OpenJDK](http://openjdk.java.net/projects/jdk/16/).
+
+### AdoptOpenJDK for Alpine Linux With musl libc 
+
+[Alpine Linux](https://alpinelinux.org/) is a popular Linux distribution for container workloads because of its small footprint. Contrary to most other Linux distributions, Alpine Linux is not based on the [C library created by the GNU project](https://www.gnu.org/software/libc/) (usually referred to as "glibc") but uses [musl libc](http://musl.libc.org) instead. So far, OpenJDK has not supported musl libc but only glibc. Therefore, we had to add the GNU C library to our container images based on Alpine Linux, which increased the container images' size. With AdoptOpenJDK 16, this is no longer necessary. We now have separate variants of AdoptOpenJDK 16 that are purpose-built for musl libc that can be downloaded as a tarball and are also available as ready-made container images.  
+
+### Preview of AdoptOpenJDK for Windows on ARM<a name="adoptopenjdk-for-windows-on-arm"></a>
+
+Our friends from Microsoft, led by our very own [Martijn Verburg](https://twitter.com/karianna), have ported [OpenJDK to Windows on ARM](https://openjdk.java.net/jeps/388) so that you can run Java (and Java-based applications) natively on your [Surface Pro X](https://www.microsoft.com/en-us/p/surface-pro-x/8qg3bmrhnwhk) or the Lenovo IdeaPad 5G. Therefore, AdoptOpenJDK 16 is now available for Windows on ARM, too, but only as a preview. The reason is that Windows on ARM is a relatively new thing with minimal hardware and software availability. Therefore, we have not yet been able to test it on a regular schedule like we usually do.
+
+### Removal of 10 Symantec Root Certificates
+
+As part of the general distrust of Symantec root certificates, 10 of them have been removed from AdoptOpenJDK. For details, see [the related issue over at the Mozilla Foundation](https://bugzilla.mozilla.org/show_bug.cgi?id=1670769).
+
+### Removal of XL Variants of OpenJ9
+
+So far, AdoptOpenJDK with Eclipse OpenJ9 came in two flavors: "normal" and "XL". The XL variant was purpose-built for workloads with very large heaps (hence the name). Thanks to recent improvements to OpenJ9, this distinction is no longer necessary. [AdoptOpenJDK 16 with Eclipse OpenJ9 is available as a single build that automatically switches between the normal and XL mode](https://eclipse.github.io/openj9-docs/version0.25/#single-build-for-compressed-references-and-non-compressed-references) depending on the configured heap size (`-Xmx` switch). If the heap size is larger than 57 GB, it will automatically enable non-compressed references, which corresponds to the old XL mode. On AIX and Linux with the metronome garbage collection policy (`-Xgcpolicy:metronome`), the threshold is at 25 GB instead of 57 GB. Next month, AdoptOpenJDK 8 and 11 with OpenJ9 will get the same capabilities. 
```

Comment:
> Eclipse OpenJ9 came in two flavors: "normal" and "XL"

I'm in two minds over referring to `normal` vs `XL` instead of just normal and large. This should probably have some review from the OpenJ9 team so I'll tag in @pshipton to get some input on that side in case he wants to adjust how it's marketed here :-)
Perhaps `"normal" and "large heap" (XL)`?

Reply:
Right, "Large Heap" is better since that is how the web pages refer to them, and the api uses "large".